.github/workflows/build-and-publish.yaml

```diff
@@ -15,37 +15,39 @@
 name: Build and Publish FMS-hf-tuning Library
 
 on:
-    workflow_dispatch:
-        inputs:
-            release-version:
-              type: string
-              required: true
-              description: 'Version number (for example: 0.0.1)'
+    release:
+      types: [published]
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          # NOTE: TGIS does not support 3.8 or 3.11
```

Comment:
I believe TGIS does support `3.11` ..  `3.8` is deprecated, so we don't need to support that.

Reply:
Thanks @gkumbhat... I'll add 3.11.  I've made a lot of changes and it's stuck failing test ATM, trying to figure out what's wrong...

Reply:
OK - added, thanks!